### PR TITLE
Increased accuracy of timer by using date, added makefile

### PR DESCRIPTION
**Pull request by AlexBocken**
I thought you might appreciate this change I made on my fork.
Instead of relying that the routine in `pomodoro()` stays at roughly 1 second it now always checks the planned end time against the current time.
Also, I added a Makefile for ease of install.

### CODE REVIEW

File: muccadoro [hunk 1]
```diff
@@ -184,6 +184,8 @@ pomodoro() {
       # <http://mywiki.wooledge.org/SignalTrap#When_is_the_signal_handled.3F>.
       sleep 1 &
       # See <http://mywiki.wooledge.org/BashFAQ/002>.
+      planned_end_time=$(( $start_time_secs + $duration ))
+      seconds=$(( $planned_end_time - $( date +'%s')  ))
```
Comment (meribold) [inline muccadoro]:
```suggestion
      planned_end_time=$((start_time_secs + duration))
      seconds=$((planned_end_time - $(date +'%s')))
```

File: muccadoro [hunk 2]
```diff
@@ -206,7 +208,7 @@ pomodoro() {
          return $fail
       fi
       wait
-      ((--seconds <= 0)) && return 0
+      (( --seconds <= 0 )) && return 0
```
Comment (meribold) [inline muccadoro]:
```suggestion
      ((--seconds <= 0)) && return 0
```

**Review by meribold — CHANGES_REQUESTED** [review]
That *is* better, thanks! I personally don't think that we need a makefile, though, and removed it from the pull request.

**Comment by AlexBocken** [comment]
Alright, I thought that might be an issue with all the methods already available with your build.
Thanks for the merge!

**Pull request merged**